generative_ai/embedding_model_tuning.py

```diff
@@ -34,7 +34,7 @@ def tune_embedding_model(
     batch_size: int = 50,
     iterations: int = 300,
 ) -> pipeline_jobs.PipelineJob:
-    match = re.search(r"(.+)(-autopush|-staging)?-aiplatform.+", api_endpoint)
+    match = re.search(r"([\w]+-[\w]+).*-aiplatform.+", api_endpoint)
     location = match.group(1) if match else "us-central1"
```

Comment:
nit: consider a bit more global friendly location (unless your API is location restricted) for non-US folks as well

Reply:
I guess that 99.9% of the Vertex AI samples would use "us-central1" as the baseline.

Reply:
Just a friendly reminder that not everyone needs to stick to one region, and there is a cost/environmental concerns over sticking only to one location. Fine to keep as-is, but if there's no restriction consider using different locations in the future.